applications/asset_tracker_v2/prj_nrf9160dk_nrf9160ns.conf

```diff
@@ -0,0 +1,107 @@
+#
```

Comment:
please merge the common content of `prj_nrf9160dk_nrf9160ns.conf` and `prj_thingy91_nrf9160ns.conf` into a `prj.conf` and then create board fragments for the differences needed under:
`boards/nrf9160dk_nrf9160ns.conf` and `boards/nrf9160dk_nrf9160ns.conf` as described here:
https://docs.zephyrproject.org/latest/guides/kconfig/setting.html#the-initial-configuration